test-d/foundry/foundry.js/clientDocuments/item.test-d.ts

```diff
@@ -57,5 +57,4 @@ if (item) {
   expectType<boolean>(item.isOwned);
   expectType<ActiveEffect[]>(item.transferredEffects);
   expectType<'weapon' | 'armor'>(item.type);
-  expectType<(object & ArmorDataPropertiesData) | (object & WeaponDataPropertiesData)>(item.getRollData());
```

Comment:
I'm not happy about removing this test but because `declare global` leaks between files I'm not sure it can be fixed.

Reply:
yeah, there is not really a way around this, but I would just move the test for `Item#getRollData` into this file anyways.